.github/workflows/unit_tests.yml

```
@@ -52,7 +52,9 @@ jobs:
       - name: Run unit tests with code coverage reporting
         run: uv run pytest --cov=undate
         if: ${{ matrix.python == env.COV_PYTHON_VERSION }}
-      - name: Upload test coverage to Codecov
-        uses: codecov/codecov-action@v3
+      - name: name: Upload coverage to Codecov
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 55: mapping values are not allowed in this context

(syntax-check)